lms/static/scripts/frontend_apps/components/Dialog.js

```diff
@@ -70,12 +77,7 @@ export default function Dialog({
         style={{ zIndex: zIndexScale.dialogBackground }}
       />
       <div className="Dialog__container" style={{ zIndex: zIndexScale.dialog }}>
-        <div
-          className={classNames({
-            Dialog__content: true,
-            [contentClass]: true,
-          })}
-        >
+        <div className={classNames(contentClasses)}>
```

Comment:
While the above code works, you could make things more concise by omitting it and just using:

```
<div className={classnames('Dialog__content', contentClass)}>
```

Reply:
hmm, good point. even better, no classnames

`<div className={'Dialog__content' + (contentClass ? ` ${contentClass}` : '')}>`